### DISCUSSION TITLE
Password Hashing

### DISCUSSION THREAD

**Discussion by XHiddenProjects** [WebCrew/All-purpose-CMS-theme-editor#2] [General]
I saw a line of code stating that password you must have a password hashed by
```php
if(md5($_POST['password']) == '5f4dcc3b5aa765d61d8327deb882cf99'){
```

instend of that have a XML file called _"Credentals.xml"_
Set context as:
```xml
<?xml version="1.0"?>
<psw>{Your password}</psw>
```
then you can just do this 
```php
if(md5($_POST['password']) == md5(simplexml_load_file(path_to_Credentals.xml)->psw)){
```

For extra security you can add this _.htaccess_ code
```htaccess
<Files "Credentals.xml">  
  Order Allow,Deny
  Deny from all
</Files>
```

This is just to make things easier instead of having to hash and go looking for the code.

**Comment by WebCrew** [comment]
Closed because of privacy and also no longer relevant.
